### PR TITLE
Fixes & New Path

### PR DESCRIPTION
**Pull request by yPiuro**
Fixed Issue [cross-compile cannot find artifact #15](https://github.com/Notliam99/TestingApi/issues/15)
 * [Artifact name change in settings](https://github.com/Notliam99/TestingApi/pull/16/files#diff-e94c713e2a0f35d4834b3d87ec089758ba56edc0c433d218e9e33653a59202d5)

Additions
 + [Added hello/qparams path](https://github.com/Notliam99/TestingApi/pull/16/files#diff-367c9b98e4bffae47110dd86a6318c870885c75408535750b91ff1630cfbc87b)
   Simple Query Parameter test path

Changes
 * [Removed some redundant code in args/mod.rs](https://github.com/Notliam99/TestingApi/pull/16/files#diff-c5354523dab7ee14c84278715c952cb7e302af46f5c8372b94375b36f003b93e)

### CODE REVIEW

**Comment by yPiuro** [comment]
Review and merge when approved, sorry for the little delay.

File: .github/workflows/cross-compile.yaml [hunk 1]
```diff
@@ -10,24 +10,24 @@ jobs:
           - release_for: Linux-x86_64
             os: ubuntu-20.04
             target: x86_64-unknown-linux-gnu
-            bin: rust_grade
```
Comment (Notliam99) [inline .github/workflows/cross-compile.yaml:13]: funny mistake

**Review by Notliam99 — CHANGES_REQUESTED** [review]
Simple fix

**Review by Notliam99 — APPROVED** [review]
Simple fix nice!

**Review by Ellozac — APPROVED** [review]
yippe

**Pull request merged**